modules/machineset-yaml-azure.adoc

```diff
@@ -18,29 +30,56 @@ kind: MachineSet
 metadata:
   labels:
     machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+ifndef::infra[]
     machine.openshift.io/cluster-api-machine-role: <role> <2>
     machine.openshift.io/cluster-api-machine-type: <role> <2>
   name: <infrastructureID>-<role>-<region> <3>
+endif::infra[]
+ifdef::infra[]
+    machine.openshift.io/cluster-api-machine-role: infra <2>
+    machine.openshift.io/cluster-api-machine-type: infra <2>
+  name: <infrastructureID>-infra-<region> <3>
+endif::infra[]
   namespace: openshift-machine-api
 spec:
   replicas: 1
   selector:
     matchLabels:
       machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+ifndef::infra[]
       machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role>-<region> <3>
+endif::infra[]
+ifdef::infra[]
+      machine.openshift.io/cluster-api-machineset: <infrastructureID>-infra-<region> <3>
+endif::infra[]
   template:
     metadata:
       creationTimestamp: null
       labels:
         machine.openshift.io/cluster-api-cluster: <infrastructureID> <1>
+        ifndef::infra[]
```

Comment:
seems that this line is not needed, it looks like this from the preview doc
```
        machine.openshift.io/cluster-api-cluster: <infrastructureID> 
        ifndef::infra[]
        machine.openshift.io/cluster-api-machine-role: <role> 
        machine.openshift.io/cluster-api-machine-type: <role> 
        machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role>-<region> 
        endif::infra[]
        ifdef::infra[]
        machine.openshift.io/cluster-api-machine-role: infra 
        machine.openshift.io/cluster-api-machine-type: infra 
        machine.openshift.io/cluster-api-machineset: <infrastructureID>-infra-<region> 
        endif::infra[]
```

Reply:
@sunzhaohua2 Are you suggesting that we remove the "hard-coded" `infra` in the examples? 

Remove:
```
machine.openshift.io/cluster-api-machine-type: infra 
machine.openshift.io/cluster-api-machineset: <infrastructureID>-infra-<region>
```
And go with this instead:

```
machine.openshift.io/cluster-api-machine-role: <role> 
machine.openshift.io/cluster-api-machine-type: <role> 
machine.openshift.io/cluster-api-machineset: <infrastructureID>-<role>-<region>  

```
I'm new to this project, but I like having the `infra` in the example if it is required. I can defer to your opinion here. 
Or, better, should we wait for Cody to return and revisit this issue?

Reply:
@mburke5678 Sorry for the misunderstanding, I mean the format of this yaml file is abnormal in the review doc, the template part should be like below:
https://deploy-preview-32296--osdocs.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#machineset-yaml-azure_creating-infrastructure-machinesets
```
  template:
    metadata:
      creationTimestamp: null
      labels:
        machine.openshift.io/cluster-api-cluster: <infrastructureID> 
        machine.openshift.io/cluster-api-machine-role: infra 
        machine.openshift.io/cluster-api-machine-type: infra 
        machine.openshift.io/cluster-api-machineset: <infrastructureID>-infra-<region> 
    spec:
```

Reply:
@sunzhaohua2 I understand. Thank you for the clarification.